pages/blog/building-graphql-servers-in-2022.mdx

```diff
@@ -0,0 +1,55 @@
+import withArticle from '../../ui/blog/article'
+
+export const meta = {
+  title: 'Building GraphQL Servers in 2022',
+  author: 'notrab',
+  tags: ['graphql', 'conference', 'yoga'],
+  date: '2022-06-28',
+  description:
+    "In this talk we'll learn about the tools needed to build your own GraphQL server from scratch using GraphQL Yoga.",
+  image:
+    '/blog-assets/building-graphql-servers-in-2022/building-graphql-servers-in-2022.png',
+  thumbnail:
+    '/blog-assets/building-graphql-servers-in-2022/building-graphql-servers-in-2022-thumbnail.png'
+}
+
+export default withArticle(meta)
+
+Building GraphQL servers in 2022 couldn't be any easier thanks to GraphQL Yoga. This year at [HasuraCon](https://hasura.io/events/hasura-con-2022/building-graphql-servers-in-2022-overview-of-the-bestnew-tools-you-should-add-to-your-toolbelt/) I discussed how Yoga started, how it evolved, and what we can expect from the future.
+
+For many years building GraphQL servers in the Node ecosystem our choices have been limited. One server library has dominated the ecosystem. With it's pairing client library it was often a good fit for a very long time. However, I believe there are more mature and battle tested solutions available today, thanks to the evolution of Yoga.
```

Comment:
Maybe we can actually mention the library?

Reply:
Sure, do you want to make a suggestion?

Reply:
```suggestion
For many years of building GraphQL servers in the Node ecosystem, our choices have been limited. Apollo Server library has dominated the ecosystem. With its pairing client library, it was often a good fit for a very long time. However, I believe there are more mature and battle-tested solutions available today, thanks to the evolution of Yoga.
```